mitiq/observable/observable.py

```diff
@@ -115,13 +139,36 @@ def partition(self, seed: Optional[int] = None) -> None:
         self._ngroups = len(self._groups)
 
     def measure_in(self, circuit: QPROGRAM) -> List[QPROGRAM]:
+        """Generates measurement circuits for the observable.
+
+        Given a quantum circuit, this method returns a list of circuits where
```

Comment:
```suggestion
        """Given a quantum circuit, this method returns a list of circuits where
```

I think we can drop the first sentence. WDYT?

Reply:
Yeah sure!